src/view/UserphraseView.h

```diff
@@ -40,8 +41,9 @@ class UserphraseView final : public QListView {
 public slots:
     void showAddUserphraseDialog();
     void showModifyUserphraseDialog();
+    void showDeleteUserphraseDialog();
     void addPhrase(int result);
-    void remove();
+    void remove(int result);
```

Comment:
this is called by a signal (dialog finish) and it will send result to show whether this dialog send accept (user clicked the "OK" button) or not, if click "no" then this function will quit.